addons/xbmc.python/addon.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="xbmc.python" version="2.24.0" provider-name="Team-Kodi">
-  <backwards-compatibility abi="2.1.0"/>
+<addon id="xbmc.python" version="2.25.0" provider-name="Team-Kodi">
+  <backwards-compatibility abi="2.25.0"/>
```

Comment:
how about adding a k* repo and putting an adapted boblight addon version in it? (just thinking out loud) - indeed boblight might be the only addon in our repo which uses the render capture interface. If there is any way which only needs adaptions to that addon then it would be fine. Is versioning the capture interface only a possible solution?